message-broker/rabbitmq/receive.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
Needs to be `#!/usr/bin/env python3`.  Then you can dispense with the `python3` directive in the README.md.  This tripped me up at first b/c I missed the `python3` command.  Guess I thought it was just part of the prompt.